### PR TITLE
Admins can export a list of all Books as a CSV

### PR DESCRIPTION
**Pull request by stephaniewillhide**

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -18,4 +18,15 @@ class Book < ApplicationRecord
   scope :featured, -> { where(featured: true) }
 
   paginates_per 3
+
+  def self.to_csv
+    attributes = %w{ name isbn featured }
+    CSV.generate() do |csv|
```
Comment (jordanmaguire) [inline app/models/book.rb:24]: No need for the () at the end of `generate`. By convention in Ruby, we only provide brackets when we are passing arguments through.

The only exception to this rule is the `super` method, but we'll get into that a little ways down the track.

File: app/views/books/_form.html.erb [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+  <%= form.input(:featured, label: "Featured?", as: :boolean) %>
   <%= form.drag_and_drop_file_field(:cover) %>
   <%= form.submit %>
 <% end %>
+
+<style>
+  #book_cover {
+    z-index: -5;
+  }
+</style>
```
Comment (jordanmaguire) [inline app/views/books/_form.html.erb:13]: Is this still required? I remember we had talked about this and it was related to the form when it was being displayed horizontally.

**Review by jordanmaguire — APPROVED** [review]
Looks great 👍

File: app/views/books/_form.html.erb (hunk 2 above)
Comment (stephaniewillhide) [reply app/views/books/_form.html.erb]: Nope! Removed

**Pull request merged**